FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,19 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			
+			switch result {
+			case let .success((data, response)):
+				guard response.statusCode == 200, let images = try? RemoteFeedImageMapper.feedImages(from: data) else {
+					completion(.failure(Error.invalidData))
+					return
+				}
+				completion(.success(images))
+			case .failure(_):
```

Comment:
`(_)` is not needed, please remove it:
```suggestion
			case .failure:
```